http2/testdata/search.proto

```
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+import "testdata/common.proto";
```

Buf (1.47.2)

import "testdata/common.proto": file does not exist

(COMPILE)